src/tasks/tasks.py

```diff
@@ -1811,31 +1671,14 @@ def _make_instance(input1, input2, labels):
             yield _make_instance(sent1, sent2, trg)
 
 
-class DisSentWikiSingleTask(DisSentTask):
-    ''' Task class for DisSent with Wikitext 103 only considering clauses from within a single sentence
-        Data sets should be prepared as described in Nie, Bennett, and Goodman (2017) '''
-
-    def __init__(self, path, max_seq_len, name="dissentwiki"):
-        super().__init__(path, max_seq_len, "wikitext.dissent.single_sent", name)
-
-
-@register_task('dissentwikifullbig', rel_path='DisSent/wikitext/')
-class DisSentWikiBigFullTask(DisSentTask):
-    ''' Task class for DisSent with Wikitext 103 considering clauses from within a single sentence
-        or across two sentences.
-        Data sets should be prepared as described in Nie, Bennett, and Goodman (2017) '''
-
-    def __init__(self, path, max_seq_len, name="dissentwikifullbig"):
-        super().__init__(path, max_seq_len, "wikitext.dissent.big", name)
-
-
+# TODO: does this even work? What is n_classes for this?
```

Comment:
I think `n_classes = 8` for all DisSent tasks.
I think we didn't use `DisSentWikiSingleTask`.
Maybe @tommccoy1 can comment on if that's right, and if it's okay to delete the within-sentence task?

Reply:
Let's leave as a TODO for now so that this PR stays a no-op.